docs/content/documentation/templates/overview.md

```diff
@@ -23,7 +23,7 @@ A few variables are available on all templates except feeds and the sitemap:
 Config variables can be accessed like `config.variable`, in HTML for example with `{{ config.base_url }}`.
 The 404 template does not get `current_path` and `current_url` (this information cannot be determined).
 
-On top of the `config` attributes mentioned above, it also gets `config.mode` which is whether it's run in `build`, `serve` or `check`.
```

Comment:
It should actually be lower cased, I've forgotten to set the correct serde attribute. I've just pushed a change on next for that. If the 0.16.1 release doesn't happen too soon, I'll merge that in the meantime though.